### ISSUE TITLE
No support for timestamp?

### ISSUE THREAD

**Issue by intelligentestech** [bamcis-io/PrestoClient#23]
Looks like the type mapping doesn't include support for timestamp data type in presto? Is there a reason for that, or just an oversight? If this repo is still actively maintained, happy to provide a PR for this, but first wanted to check to see if there is a known reason why this was not included.

**Comment by hakenmt** [comment]
I don't think that data type existed when I first created this project. The client was only tested against the version specified in the branch (which is far before presto became trino). If you'd like to submit a PR, happy to take a look.

**Comment by intelligentestech** [comment]
Thanks. I did end up forking it and adding timestamp support. Works great with IBM Watsonx.data (which is still presto).

Thanks,
Chris
________________________________
